images/salt-master/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos:7.6.1810
+FROM base-centos-image
 
 MAINTAINER moonshot-platform <moonshot-platform@scality.com>
```

Comment:
MAINTAINER is deprecated, please don't use it :) https://docs.docker.com/engine/reference/builder/#maintainer-deprecated
Use a LABEL instead